rero_ils/modules/patrons/cli.py

```diff
@@ -151,3 +154,49 @@ def import_users(infile, append, verbose, password, lazy, dont_stop_on_error,
                 for line in json.dumps(error_record, indent=2).split('\n'):
                     error_file.write('  ' + line + '\n')
             error_file.write(']')
+
+
+@click.command('users_validate')
+@click.argument('jsonfile', type=click.File('r'))
+@click.option('-v', '--verbose', 'verbose', is_flag=True, default=False)
+@click.option('-d', '--debug', 'debug', is_flag=True, default=False)
+@with_appcontext
+def users_validate(jsonfile, verbose, debug):
+    """Check users validation."""
+    click.secho('Validate user file', fg='green')
+
+    path = current_jsonschemas.url_to_path(get_schema_for_resource('ptrn'))
+    ptrn_schema = current_jsonschemas.get_schema(path=path)
+    ptrn_schema = records_state.replace_refs(ptrn_schema)
+    # TODO: get user schema path programaticly
+    # path = current_jsonschemas.url_to_path(get_schema_for_resource('user'))
+    path = 'users/user-v0.0.1.json'
+    user_schema = current_jsonschemas.get_schema(path=path)
+    user_schema = records_state.replace_refs(user_schema)
+
+    merger_schema = {
+        "properties": {
+            "required": {"mergeStrategy": "append"}
+        }
+    }
+    merger = Merger(merger_schema)
+    schema = merger.merge(user_schema, ptrn_schema)
```

Comment:
Did you already tried the simple python way : `dict_z = {**dict_x, **dict_y}` ? For python 3.9, it's even easier :: `dict_z = dict_x | dict_y`

Reply:
Yes, but it is not merging the required correctly.